### PR TITLE
Fix: #85

### PR DESCRIPTION
**Pull request by ralmsdeveloper**
Resolve: https://github.com/ErikEJ/EFCorePowerTools/issues/85#issuecomment-415371030

cc/ @ErikEJ 

I believe that next month I will be able to add tests for the project in the area of ​​Migrations

### CODE REVIEW

File: src/GUI/EFCorePowerTools/EFCorePowerTools.csproj [hunk 1]
```diff
@@ -160,6 +166,10 @@
       <Link>efpt21.exe.zip</Link>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <Content Include="efpt\2.1.2\efpt.exe.config">
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </Content>
+    <None Include="app.config" />
```
Comment (ErikEJ) [inline src/GUI/EFCorePowerTools/EFCorePowerTools.csproj]: remove

File: src/GUI/EFCorePowerTools/Handlers/ProcessLauncher.cs [hunk 2]
```diff
@@ -133,13 +133,14 @@ private string DropFiles(string outputPath)
...
                 if (version.ToString(3) == "2.1.0") testVersion = "2.1.0";
+                if (version.ToString(3) == "2.1.2") testVersion = "2.1.2";
             }
 
             File.Copy(Path.Combine(fromDir, "efpt.exe"), Path.Combine(toDir, "efpt.exe"), true);
 
             if (!string.IsNullOrEmpty(testVersion))
             {
-                if (testVersion == "2.1.0")
+                if (testVersion == "2.1.0" || testVersion == "2.1.2")
```
Comment (ErikEJ) [inline src/GUI/EFCorePowerTools/Handlers/ProcessLauncher.cs]: Should drop support for 2.0.2 and only support 2.0.0 and latest patch and 2.1.0 and latest patch - I have logged https://github.com/ErikEJ/EFCorePowerTools/issues/90

File: src/GUI/EFCorePowerTools/app.config [hunk 3]
```diff
@@ -0,0 +1,79 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ErikEJ) [inline src/GUI/EFCorePowerTools/app.config]: Delete

File: src/GUI/ReverseEngineer20/ReverseEngineer20.csproj [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\Microsoft.EntityFrameworkCore.Design.2.1.0\build\net461\Microsoft.EntityFrameworkCore.Design.props" Condition="Exists('..\packages\Microsoft.EntityFrameworkCore.Design.2.1.0\build\net461\Microsoft.EntityFrameworkCore.Design.props')" />
+  <Import Project="..\packages\Microsoft.EntityFrameworkCore.Design.2.1.2\build\net461\Microsoft.EntityFrameworkCore.Design.props" Condition="Exists('..\packages\Microsoft.EntityFrameworkCore.Design.2.1.2\build\net461\Microsoft.EntityFrameworkCore.Design.props')" />
```
Comment (ErikEJ) [inline src/GUI/ReverseEngineer20/ReverseEngineer20.csproj]: Revert upgrade (reverse engineer != modelling/migrations), and requires version support from all involved providers

File: src/GUI/ReverseEngineer20/ReverseEngineer20.csproj [hunk 5]
```diff
@@ -226,6 +226,7 @@
     <Compile Include="ReverseEngineer\ReverseEngineerOptions.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (ErikEJ) [inline src/GUI/ReverseEngineer20/ReverseEngineer20.csproj]: Remove

File: src/GUI/ReverseEngineer20/app.config [hunk 6]
```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ErikEJ) [inline src/GUI/ReverseEngineer20/app.config]: Delete

File: src/GUI/ReverseEngineer20/packages.config [hunk 7]
```diff
@@ -5,43 +5,43 @@
   <package id="Handlebars.Net" version="1.9.4" targetFramework="net461" />
   <package id="Microsoft.CSharp" version="4.5.0" targetFramework="net461" />
   <package id="Microsoft.Data.Sqlite.Core" version="2.1.0" targetFramework="net461" />
```
Comment (ErikEJ) [inline src/GUI/ReverseEngineer20/packages.config]: Revert

File: src/GUI/UnitTests/UnitTests.csproj [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ErikEJ) [inline src/GUI/UnitTests/UnitTests.csproj]: Revert

File: src/GUI/UnitTests/app.config [hunk 9]
```diff
@@ -0,0 +1,79 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ErikEJ) [inline src/GUI/UnitTests/app.config]: Delete

File: src/GUI/UnitTests/packages.config [hunk 10]
```diff
@@ -1,42 +1,43 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.CSharp" version="4.4.0" targetFramework="net461" />
```
Comment (ErikEJ) [inline src/GUI/UnitTests/packages.config]: Revert all

**Comment by ralmsdeveloper** [comment]
@ErikEJ  thanks for your time in review, I will prepare a better PR, just made some adjustments, to generate migrations we need to have singular versions of the core, so I will be preparing a better PR, adding comments. I am interested in helping, I have friends who need to use their tools and I found it fantastic.

**Comment by ErikEJ** [comment]
Will be fixed in #91

**Pull request closed without merging**